### ISSUE TITLE
[Feature] Update entity for stacks

### ISSUE THREAD

**Issue by dd310** [tomaae/homeassistant-portainer#8]
## Is your feature request related to a problem? Please describe.

It would be nice to have a way to monitor the availability of a new image for a stack/container and trigger the update from Home Assistant.

## Describe the solution you'd like

I don't know if it's feasible, but maybe it could be possible to introduce an update. entity in Home Assistant that senses whether or there's a new image available and sends a POST request to a specific stack webhook, triggering the image pull and the stack re-deployment.

**Comment by tomaae** [comment]
As far as I know, docker in not aware of available update. You normally have to use watchtower.

**Comment by dd310** [comment]
Currently I'm using WUD (https://github.com/fmartinou/whats-up-docker) and mqtt to receive a notification on HA when a new image is available. However, since it does not integrate with Portainer API, i still have to log in the instance and re-deploy manually.

I noticed there's a little dot next to the stack name in the portainer GUI that turns red when an image update is available, but I can't say if it could be useful.

![Screenshot_20230714_002443_Chrome.jpg](https://github.com/tomaae/homeassistant-portainer/assets/17815448/f5ddc988-b840-49c8-96dd-3f39daf30560)

**Comment by tomaae** [comment]
sure, I can check that

**Comment by solidminds** [comment]
Any update in this?

**Comment by tomaae** [comment]
No, I will be looking at it when I will be working on next feature update for this integration.
